src/d_loop.c

```diff
@@ -728,7 +728,7 @@ void TryRunTics (void)
 
     // [AM] If we've uncapped the framerate and there are no tics
     //      to run, return early instead of waiting around.
-    #define return_early (uncapped && counts == 0 && leveltime > oldleveltime)
+    #define return_early (uncapped && counts == 0)
```

Comment:
This will also affect intermission screens, finales, etc. Does this have any negative impact on these or is the general direction now "enable vsync"?

Reply:
Those screens will run at a higher framerate if vsync is disabled, sure. The default Woof settings have vsync enabled, so the fps will be limited to the refresh rate (typically 60 - 144 Hz). If a user disables vsync in Woof and doesn't set a frame rate limit, then it's up to the video driver vsync setting (auto, forced on, force off, fast, etc.). I don't think there are users that rely on the current 35 fps cap for menu/intermission/finale but then disable all the other ways to limit fps. If you want to play it safe, we can set the default limit to 60 fps, but I don't like that option.

Reply:
I like this change, I copied it to the any_res branch, everything seems to work fine.

Reply:
Do we merge or close this since it's in the any_res branch?

Reply:
I don't know. It isn't strictly related to the any_res branch, so better keep it separate @rfomin ?

Reply:
I think we can merge this. `any_res` still not finished.